Classes/Issues/Comments/QuotedComment.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  QuotedComment.swift
+//  Freetime
+//
+//  Created by Joan Disho on 09.05.18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+class QuotedComment {
+    private let feed: Feed
+    private let commentModel: IssueCommentModel
+
+    init(in feed: Feed, with commentModel: IssueCommentModel) {
```

Comment:
I feel like we shouldn't rename these external parameters in this `init`; it makes it feel like a function, which may be unintended.